src/mpi/coll/bcast/bcast_intra_binomial.c

```diff
@@ -22,8 +22,12 @@ int MPIR_Bcast_intra_binomial(void *buffer,
     int mpi_errno = MPI_SUCCESS;
     int mpi_errno_ret = MPI_SUCCESS;
     MPI_Aint nbytes = 0;
+#ifdef HAVE_ERROR_CHECKING
     MPI_Aint recvd_size;
     MPI_Status status;
+#else
+    MPI_Status status = *MPI_STATUS_IGNORE;
```

Comment:
Surely this will cause a segfault, no?

Reply:
Ah. yes. Sorry. I overlooked. Fixed it now.